src/screen/Council/MotionsScreen.tsx

```diff
@@ -0,0 +1,273 @@
+import {useCouncilMembers} from 'screen/Council/CouncilScreen';
+import {useQuery, useQueryClient} from 'react-query';
+import {Param, Params} from 'presentational/Params';
+
+export function MotionsScreen({navigation}: {navigation: NavigationProp<DashboardStackParamList>}) {
+  const {api} = useContext(ChainApiContext);
+  const {data, refetch, isLoading} = useQuery('motions', () => api?.derive.council.proposals());
+
+  return (
+    <Layout style={globalStyles.flex}>
+      <ScreenNavigation
+        renderTitle={() => (
+          <Text category={'s1'} style={globalStyles.monoFont}>
+            Motions
+          </Text>
+        )}
+        accessoryLeft={
+          <TopNavigationAction onPress={navigation.goBack} icon={(p) => <Icon {...p} name={'arrow-back-outline'} />} />
+        }
+      />
+      <FlatList
+        refreshing={isLoading}
+        onRefresh={refetch}
+        style={styles.flatList}
+        data={data}
+        renderItem={({item}) => {
+          return <Motion item={item} />;
+        }}
+        ItemSeparatorComponent={Divider}
+        keyExtractor={(item) => item.hash.toHex()}
+        ListEmptyComponent={EmptyView}
+      />
+    </Layout>
+  );
+}
+
+const styles = StyleSheet.create({flatList: {padding: standardPadding * 2}});
+
+function Motion({item}: {item: DeriveCollectiveProposal}) {
+  const theme = useTheme();
+  const {api} = useContext(ChainApiContext);
+  const {start} = useContext(TxContext);
+  const {accounts} = useContext(AccountContext);
+  const account = accounts?.[0];
+
+  const {votes, proposal, hash} = item;
+  const {members, isMember} = useMembers();
+  const {isCloseable, isVoteable} = useVotingStatus(votes, members.length, 'council');
+
+  const {meta, method, section} = proposal.registry.findMetaCall(proposal.callIndex);
+  const params = useParams(proposal);
+
+  const [open, setOpen] = useState(false);
+  const queryClient = useQueryClient();
+
+  return (
+    <View style={motionStyle.container}>
+      <View style={motionStyle.mainRow}>
+        <Text category={'h4'}>{formatNumber(votes?.index)}</Text>
+        <TouchableOpacity onPress={() => setOpen(!open)} style={motionStyle.titleContainer}>
+          <Text category={'p1'} style={motionStyle.title} numberOfLines={1}>
+            {section}.{method}
+          </Text>
+          <Icon
+            name={open ? 'arrow-up-outline' : 'arrow-down-outline'}
+            style={globalStyles.icon}
+            fill={theme['color-basic-600']}
+          />
+        </TouchableOpacity>
+        {/*<Text>{formatNumber(votes?.threshold)}</Text>*/}
+        <Text category={'c1'}>{`Aye ${votes?.ayes.length}/${members.length} `}</Text>
+        <Padder scale={0.5} />
+        {(() => {
+          if (isMember) {
+            if (isCloseable) {
+              return (
+                <View>
+                  <Button
+                    status={'warning'}
+                    size={'tiny'}
+                    onPress={() => {
```

Comment:
nit: can we extract these functions to make the component easier to read?  e.g `onPress={onPressClose}`

Reply:
5703856 ✅